.distignore

```diff
@@ -12,4 +12,8 @@ DOCKER_ENV
 docker_tag
 Dockerfile-php-build
 output.log
-phpcs.xml
\ No newline at end of file
+phpcs.xml
+tests
```

Comment:
This should only match the top level folder:
```suggestion
/tests
```
Let’s move the entry to the top where folders are listed.

Would be great if the entries were sorted alphabetically 🫠 (or grouped in a meaningful way)